doc/source/whatsnew/v0.21.0.txt

```diff
@@ -116,6 +116,8 @@ Other API Changes
 Deprecations
 ~~~~~~~~~~~~
 - :func:`read_excel()` has deprecated ``sheetname`` in favor of ``sheet_name`` for consistency with ``.to_excel()`` (:issue:`10559`).
+- :func:`Series.argmax` has been deprecated in favor of :func:`Series.idxmax` (:issue:`16830`)
```

Comment:
Can you go into a lot more detail here? Maybe explain a bit about what's going on, how to avoid warnings. You can make your own mini-section. I haven't been following the discussion so this may be out of date, but something like

    For compatibility with NumPy arrays, `pd.Series` implements `argmax` and
    `idxmax`. Since pandas 0.13(is this correct?) :func:`Series.argmax` and
    :func:`Series.argmin` were simply aliased to `:func:Series.idxmax` and
    `:func:Series.idxmin`, which return the *label* of the maximum and minimum,
    rather than the *position*.

    Since returning the positional `argmax` and `argmin`  occasionally useful,
    we've deprecated the current behavior of :func:`Series.argmax` and
    :func:`Series.argmin`. Using either of these will emit a ``FutureWarning``.
    If you were using ``Series.argmin`` and
    ``Series.argmax``, to get the label of the minimum or maximum, switch to using
    ``idxmin`` and ``idxmax``. In the future, ``Series.argmin`` will return the
    *position* of the minimum. Likewise for ``Series.argmax``. 

    .. note::

      Due to how numpy handles method dispatch to array-like objects,
      calling ``np.argmin(Series)`` will emit a ``FutureWarning``, even though this
      is the currently recommend way of getting the position of the minimum. To
      silence this warnings, we recommend using the ``catch_warnings`` context
      manager.

      .. ipython:: python

          import warnings

          s = pd.Series([3, 1, 2])
          with warnings.catch_warnings():
              result = np.argmin(s)
          result